content/blog/hpe-oneview-and-servicenow-integration-with-stackstorm.md

```diff
@@ -0,0 +1,180 @@
+---
+title: HPE OneView and ServiceNow integration with Stackstorm
+date: 2021-09-29T17:53:58.637Z
+author: Rick Kauffman
+authorimage: /img/Avatar2.svg
+---
+![](http://www.techworldwookie.com/blogpost/flowchart.png)
```

Comment:
There is this http link that needs to be replaced by https. Also I think it would be more logical to have the picture below a first text section. No?